scripts/publish.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+REGISTRY="${ARTIFACTORY_URL}/api/npm/npm-okta"
+
+export TEST_SUITE_TYPE="build"
+
+if [ -n "$action_branch" ];
+then
+  echo "Publishing from bacon task using branch $action_branch"
+  TARGET_BRANCH=$action_branch
+else
+  echo "Publishing from bacon testSuite using branch $BRANCH"
+  TARGET_BRANCH=$BRANCH
+fi
+
+if ! npm run ci-update-package -- --branch ${TARGET_BRANCH}; then
```

Comment:
It looks like this was missed when we removed the optionalDependencies, but since we're installing ci-update-package & ci-pkg-info globally now, two things:

1. Remove these lines from package.json: https://github.com/okta/okta-signin-widget/blob/master/package.json#L41-L42

2. Here, you no longer need to do an "npm run", just call `ci-update-package --branch ${TARGET_BRANCH}` and `ci-pkginfo -t dataload etc` below